universitat-basel-iberoromanistik.csl

```diff
@@ -0,0 +1,300 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="es-ES" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>Universität Basel- Iberoromanistik (Spanish)</title>
+    <id>http://www.zotero.org/styles/universitat-basel-iberoromanistik</id>
+    <link href="http://www.zotero.org/styles/universitat-basel-iberoromanistik" rel="self"/>
+    <link href="http://www.zotero.org/styles/universitat-bremen-institut-fur-politikwissenschaft" rel="template"/>
+    <link href="http://www.politik.uni-bremen.de/ccm/cms-service/download/asset/Kompendium.pdf?asset_id=914048" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien, PhD</name>
+      <email>obrienpat86@gmail.com</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="humanities"/>
+    <updated>2018-09-28T08:05:13+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="es">
+    <terms>
+      <term name="et-al">et al.</term>
+      <term name="accessed">fecha de consulta</term>
+    </terms>
+  </locale>
+  <macro name="editor">
+    <choose>
+      <if type="chapter article-journal" match="any">
+        <names variable="editor" delimiter="/" suffix=",">
+          <name delimiter=" / " delimiter-precedes-last="always" name-as-sort-order="all"/>
+          <et-al prefix=" (" suffix=") " font-style="italic"/>
+          <label form="short" prefix=" (" suffix=")"/>
+        </names>
+      </if>
+      <else>
+        <names variable="editor" delimiter="/">
+          <name name-as-sort-order="all" sort-separator=", " delimiter="/" delimiter-precedes-last="always" form="long"/>
+          <label form="short" prefix=" (" suffix=")"/>
+        </names>
+      </else>
+    </choose>
+  </macro>
+  <macro name="anon">
+    <text term="anonymous" form="short"/>
+  </macro>
+  <macro name="author">
+    <names variable="author" delimiter=" /">
+      <name delimiter=" / " delimiter-precedes-last="always" name-as-sort-order="all"/>
+      <label form="short" prefix=" (" suffix=")"/>
+      <substitute>
+        <text macro="editor"/>
+        <text macro="anon"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author" delimiter="/">
+      <name form="short" delimiter=" / " delimiter-precedes-last="always" initialize-with=". "/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
```

Comment:
should correspond to `author` macro, so either add substitute translator there or remove here.